configs/tiny/OLMo-150M.yaml

```diff
@@ -56,20 +56,16 @@ optimizer:
 
 scheduler:
   name: cosine_with_warmup
-  units: tokens
-  t_warmup: 4194304000
-  t_max: 3e12
+  t_warmup: 5000
```

Comment:
For normal init, this is a lot of warmup? Not a big deal, but unusual?

Reply:
smaller models, higher LR, did not take a chance! never bad doing a longer warmup!